sdk/helper/ldaputil/connection.go

```diff
@@ -14,7 +14,7 @@ import (
 // but through an interface.
 type Connection interface {
 	Bind(username, password string) error
-	Close()
+	Close() error
```

Comment:
Is this change to the interface going to impact anyone? Do we need to call this out somewhere?

Reply:
Compilation errors will occur if someone tries to build with the incorrect interface, so it's kind of self documenting. Also I think this should be OK because our ldap integrations use the SDK, which is being upgraded here.